models/transactions.go

```diff
@@ -0,0 +1,137 @@
+package models
+
+import (
+	"database/sql"
+	"log"
+	"time"
+)
+
+type Transaction struct {
+	ID    string `json:"id"`
+	Lines []*TransactionLine
+}
+
+type TransactionLine struct {
+	AccountID string `json:"account"`
+	Delta     int    `json:"delta"`
+}
+
+func (t *Transaction) IsValid() bool {
+	if len(t.Lines) == 0 {
```

Comment:
@sudhirj  A transaction with no lines will be treated as Valid, if we don't make this check. Is that the expected behaviour ?